doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,257 @@
+.. _nrf_bm_release_notes_080:
+
+Release Notes for |BMlong| v0.8.0
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|BMshort| v0.8.0 is the public launch release of this product, demonstrating a set of samples for the `nRF54L15 DK`_ with build targets supporting the following SoCs:
+
+* `nRF54L15`_
+* `nRF54L10`_ (emulated on the nRF54L15 DK)
+* `nRF54L05`_ (emulated on the nRF54L15 DK)
+
+The supported DK version is `nRF54L15 DK v0.9.3`_.
+
+The release includes a prototype version of the S115 SoftDevice - a proprietary Bluetooth® LE stack to be used on the nRF54L15 DK.
+
+Product maturity
+****************
+
+As this is version 0.8.0 of |BMshort|, please note that the solution is still under development and not production ready.
+While most modules and libraries are considered stable and are unlikely to undergo significant changes, adjustments might still be necessary.
+This includes potential modifications to the organization of the folder structure.
+Although the memory partition structure is expected to remain consistent, the sizes of individual partitions might be refined as progress is made towards a more stable release.
+
+The documentation for |BMshort| is still in development and might not cover all libraries and modules available in the codebase.
+Some sections might be incomplete or lacking detailed information as content is being developed.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |BMshort| v0.8.0.
+See the :ref:`install_nrf_bm` section for more information about supported operating systems and toolchain.
+
+Supported boards
+****************
+
+* PCA10156 (`nRF54L15 DK`_)
+
+Migration from nRF5 SDK
+***********************
+
+For guidelines on how to migrate an application built on nRF5 SDK to |BMshort|, refer to `Migration notes - nRF5 SDK to nRF Connect SDK Bare Metal`_.
```

Comment:
some links do not work (404 Page Not Found), I guess content will be added soon?

Reply:
yes, as soon as we tag